### PR TITLE
ncspot: update to 0.13.1.

### PR DESCRIPTION
**Pull request by lukas-jo**
#### Testing the changes
- I tested the changes in this PR: **YES**

CC Maintainer: @ram02z

### CODE REVIEW

File: srcpkgs/ncspot/template [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'ncspot'
 pkgname=ncspot
-version=0.11.2
-revision=2
+version=0.13.0
+revision=1
 build_style=cargo
```
Comment (icp1994) [inline srcpkgs/ncspot/template:5]:
```suggestion
build_style=cargo
build_helper="qemu"
```

**Review by icp1994 — COMMENTED** [review]
this _should_ work

File: srcpkgs/ncspot/template [hunk 2]
```diff
@@ -11,11 +11,24 @@ short_desc="Cross-platform ncurses Spotify client"
 maintainer="Omar Zeghouani <omarzeghouanii@gmail.com>"
 license="BSD-2-Clause"
 homepage="https://github.com/hrkfdn/ncspot"
-distfiles="https://github.com/hrkfdn/ncspot/archive/v${version}.tar.gz"
-checksum=0fd9696e13912b906605e9972bf3b2a8a68688cce17478353713550801c94fab
+changelog="https://github.com/hrkfdn/ncspot/releases"
+distfiles="https://github.com/hrkfdn/ncspot/archive/refs/tags/v${version}.tar.gz"
+checksum=35210f8237a20eb894dc6b7cc517e60c82c2e93ff5940c267fe0bf673eccc411
+
```
Comment (icp1994) [inline srcpkgs/ncspot/template:18]:
```suggestion
post_build() {
	cargo auditable build --release --target ${RUST_TARGET} --package xtask
	vtargetrun target/${RUST_TARGET}/release/xtask generate-shell-completion
	vtargetrun target/${RUST_TARGET}/release/xtask generate-manpage
}

```

**Comment by lukas-jo** [comment]
looks and works better than what I did, thanks

**Review by ram02z — APPROVED** [review]
LGTM

**Comment by lukas-jo** [comment]
bumped to 0.13.1

**Pull request merged**